cscs-checks/tools/jupyter/jupyter_tests.py

```diff
@@ -18,5 +18,24 @@ def __init__(self):
         self.time_limit = '1m'
         self.max_pending_time = '7m'
         self.sanity_patterns = sn.assert_found(r'nid\d+', self.stdout)
-        self.tags = {'production', 'maintenance'}
+        self.tags = {'production', 'maintenance', 'health'}
         self.maintainers = ['RS', 'TR']
+
+
+@rfm.simple_test
+class JupyterHubServerCheck(rfm.RunOnlyRegressionTest):
+    def __init__(self):
+        self.descr = 'Check JupyterHub server status and version'
+        self.valid_systems = ['daint:jupyter_gpu', 'daint:jupyter_mc',
+                              'dom:jupyter_gpu', 'dom:jupyter_mc']
+        self.valid_prog_environs = ['*']
+        self.sourcesdir = None
+        self.executable = 'curl https://jupyter.cscs.ch/hub/api/'
+        # TODO: set correct waiting time
+        self.time_limit = '1s'
```

Comment:
Do not forget to set the time limit... 😬

Reply:
@twrobinson is there a time limit that you think makes sense in the above code?

Reply:
Time limit set to 30s, as suggested by @twrobinson